docs/semgrep-app/findings.md

```diff
@@ -119,36 +149,49 @@ To **open findings**:
 1. Click **Ignored** to see all ignored findings.
 2. After the findings are filtered, perform one of these steps:
     - Select all of the results by clicking on the header row checkbox. You can navigate to succeeding pages and add to the current selection.
-    - Select relevant findings one-by-one by clicking on their checkboxes individually.
+    - Select relevant findings one by one by clicking on their checkboxes individually.
 3. Click the **Triage** button.
 4. Click **Open**.
 5. Optional: Include a **comment** explaining the action.
 
-To **close findings**, do one of the two options:
-
-* Update, fix, or refactor the code such that the Semgrep rule pattern no longer matches the code.
-* Remove the rule from the Rule Board.
+To **fix a finding**, update, or refactor the code such that the Semgrep rule pattern no longer matches the code.
 
 To **remove a rule** from the Rule Board:
 
 1. Click **Rule Board**.
-2. Click on the ruleset that contains the rule.
-3. Drag the rule tile to the **Discard rule** section.
+2. Click the ruleset that contains the rule.
+3. Click the **garbage can** icon next to the rule you're deleting.
+4. Click ****Save**.
 
 To **view comments**:
 
 * Click the **speech balloon** next to the finding’s status.
 
 ## Creating Jira tickets from findings
 
-Semgrep supports the creation of Jira tickets from a finding. This feature is available to Team/Enterprise Tier users.
+Semgrep supports the creation of Jira tickets from a finding. This enables developers and project managers to create relevant issues within their own project or bug tracking environment. This feature is available to Team/Enterprise Tier users.
 
 To **create a ticket**:
 
 1. Set up a Jira integration through the [Integrations](https://semgrep.dev/docs/semgrep-app/integrations/) guide.
 2. Click on the **three-dot icon** of the entry.
 3. Click **Create issue with Jira**.
 
+## Deduplicating findings
+
+Semgrep App scans are performed on both mainline (trunk) and non-mainline branches. Findings are generated on a per-branch basis. Duplicate findings arise due to scans occurring on the same code in different branches.
+
+There are two types of scans:
+
+<dl>
+    <dt>Full scan</dt>
+    <dd>Scans the repository in its entirety. It is recommended to perform full scans on mainline branches, such as `master` or `main`. This scan is performed on a scheduled basis.</dd>
+    <dt>Diff-aware scan</dt>
+    <dd>Diff-aware scans are performed on non-mainline branches, such as in pull requests and merge requests. Diff scans traverse the repository's files based on the commit where the branch diverged from the mainline branch (or diverged from the last commit that was fully scanned?)</dd>
+</dl>
+
+If a finding is fixed in one branch (such as `main`) but open in another (such as `production`), and the code fixes are present in both branches, initiate a scan on the branch(es) with open findings to mark the findings as fixed.
```

Comment:
Would Semgrep mark these open findings as now fixed, or does the user have to mark them? I’m not quite sure from the last phrase.

Reply:
Semgrep will mark the findings as fixed. I will add this in.